### ISSUE TITLE
When switching to visual stepping mode block editor doesn't update

### ISSUE THREAD

**Issue by dekrain** [jmoenig/Snap--Build-Your-Own-Blocks#1536]
...if there are open dialogs.

**Comment by jmoenig** [comment]
Yep. We currently don't have backpointers from the IDE to open block editors, i.e. we don't register open block editors with the IDE, and I'm reluctant to do so for a variety of reasons. Let's think about this!
